Data/ByteString/Lazy.hs

```diff
@@ -563,29 +578,25 @@ compareLength (Chunk c cs) toCmp  = compareLength cs (toCmp - fromIntegral (S.le
    n <= length t = compareLength t n /= LT
   #-}
 
+mapAccumLChunks :: (acc -> S.ByteString -> (acc, S.ByteString)) -> acc -> ByteString -> (acc, ByteString)
+mapAccumLChunks function accumulator = fmap (L.foldr Chunk Empty) . L.mapAccumL function accumulator . foldrChunks (:) [ ]
```

Comment:
IMHO the long argument names are

1. unconventional
2. don't really aid readability

I'd slightly prefer to have the usual `f` and `a` or similar here.

Reply:
I am not really sure about any of it.

1. There are a lot of stupid conventions out there that I would rather dodge. Apparently it is conventional to `import qualified Data.ByteString        as P`? Not every convention is worth perpetuating.

2. I am not so sure. I think it is easier to read and also easier to write the long way. See my reasons below:

   * I have been trying to learn to speak Programminglish for a while, but eventually I decided it is not worth the effort and now I write everything in plain English by default. This saves me a lot of decision effort. Should I write `function` or `f` or `fct` or `fun` or `fn` or `func` or… All of these are real examples! The thought of going through the patch and deciding for every identifier if I should make it shorter, and if so shorter by how much, and which letter would be conventional to use if it is to be a single letter… this thought fills me with pain.

   * I also believe that letting the reader actually read your mental model right off the code is helpful to people that are not very familiar with the mental models a narrow expert would consider to be usual, while experts can understand the code well enough either way. Yes, `function` is kind of obvious… for a person used to functions being first class entities… `accumulator` is already not so obvious.

   * [Historically, books were written with ligatures. See some Holy Bible manuscript for examples.](https://en.wikipedia.org/wiki/Scribal_abbreviation) But you would not find any ligatures like that in a modern book. I think the reason is that ligatures do not scale.

I do not want to derail everything for this reason, so if you insist I can do it your way.

Reply:
Please unresolve this if you really want me to rename things.